eras/byron/ledger/impl/cardano-ledger-byron.cabal

```diff
@@ -110,6 +110,7 @@ data-files:
     golden/cbor/utxo/TxSigData
     golden/cbor/utxo/TxWitness
     golden/json/genesis/GenesisData0_Legacy_HasNetworkMagic
+    mainnet-genesis.json
```

Comment:
You need to bump patch version of this package, otherwise it will not get released.

Also you need to add the same file to the [`cardano-ledger-byron-test` package](https://github.com/IntersectMBO/cardano-ledger/blob/9d4375abaefff1b89587157fdb124a43f542231b/eras/byron/ledger/impl/test/cardano-ledger-byron-test.cabal#L112) and bump its patch version too.

Reply:
Got it! Thanks! 👍